js/content/store.js

```diff
@@ -171,77 +171,85 @@ class StorePageClass {
 
         // Ubisoft Uplay detection
         let uplay =
-               text.includes("Uplay")
-            || text.includes("Ubisoft Account");
+               text.includes("uplay")
+            || text.includes("ubisoft account");
 
         // Securom detection
-        let securom = text.includes("SecuROM");
+        let securom = text.includes("securom");
 
         // Tages detection
         let tages =
-                text.match(/\b(tages|solidshield)\b/i)
-            && !text.match(/angebote des tages/i);
+                text.match(/\b(tages|solidshield)\b/)
+            && !text.match(/angebote des tages/);
 
         // Stardock account detection
-        let stardock = text.includes("Stardock account");
+        let stardock = text.includes("stardock account");
 
         // Rockstar social club detection
         let rockstar =
-               text.includes("Rockstar Social Club")
-            || text.includes("Rockstar Games Social Club");
+               text.includes("rockstar social club")
+            || text.includes("rockstar games social club");
 
         // Kalypso Launcher detection
-        let kalypso = text.includes("Requires a Kalypso account");
+        let kalypso = text.includes("requires a kalypso account");
 
         // Denuvo Antitamper detection
-        let denuvo = text.match(/\bdenuvo\b/i);
-
-        // Detect other DRM
-        let drm =
-                text.includes("3rd-party DRM")
-            && !text.match(/No (3rd|third)[- ]party DRM/i);
+        let denuvo = text.includes("denuvo");
 
         let drmNames = [];
-        if (gfwl) { drmNames.push('Games for Windows Live'); }
-        if (uplay) { drmNames.push('Ubisoft Uplay'); }
-        if (securom) { drmNames.push('SecuROM'); }
-        if (tages) { drmNames.push('Tages'); }
-        if (stardock) { drmNames.push('Stardock Account Required'); }
-        if (rockstar) { drmNames.push('Rockstar Social Club'); }
+        if (gfwl) { drmNames.push("Games for Windows Live"); }
+        if (uplay) { drmNames.push("Ubisoft Uplay"); }
+        if (securom) { drmNames.push("SecuROM"); }
+        if (tages) { drmNames.push("Tages"); }
+        if (stardock) { drmNames.push("Stardock Account Required"); }
+        if (rockstar) { drmNames.push("Rockstar Social Club"); }
         if (kalypso) { drmNames.push("Kalypso Launcher"); }
         if (denuvo) { drmNames.push("Denuvo Anti-tamper"); }
-        drm = drm || drmNames.length > 0;
+
+        let drm = false;
+        let drmString = "";
+        if (drmNames.length > 0) {
+            drmString += `(${drmNames.join(", ")})`;
+            drm = true;
+        } else { // Detect other DRM on app pages
+            let drmNode = document.querySelector("#category_block > .DRM_notice");
+            if (drmNode) {
+                drmString += drmNode.innerHTML.replace("<br>", ", ").trim();
+                drm = /\b(drm|account)\b/i.test(drmString);
```

Comment:
Although the text is translated by Steam, most translations kept the word "DRM", so we can match that. "Account" on the other hand is to match the custom text provided by devs.

Reply:
To give more information on how this works:
There're 3 kinds of DRM notices (or 3 templates provided by Steam), incorporates DRM, requires 3rd-party account, and EULA. Most devs only put 1 notice on their app page, some put 2: DRM+EULA or 3rd party account+EULA, rarely more, like [Starwars Jedi](https://store.steampowered.com/app/1172380/STAR_WARS_Jedi_Fallen_Order/) with a whopping 4 including a custom one.
As mentioned in the OP, they appear in order and it's usually the first one that contains DRM or 3rd party account information. For apps that have all 3 notices, well...I suppose DRM information is more important to display.